README.rst

```diff
@@ -1,225 +1,145 @@
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To contribute Superset assets:
 
-Superset supports creating virtual datasets, which are datasets defined using a SQL query instead of mapping directly to an underlying database object. Aspects leverages virtual datasets, along with `SQL templating <https://superset.apache.org/docs/installation/sql-templating/>`_, to make better use of table indexes.
+1. Fork this repository and set up a local Tutor instance with Aspects installed.
+2. Export the desired assets using Superset’s “Export” feature.
+3. Use the command:
 
-To make it easier for developers to manage virtual datasets, there is an extra step that can be done on the output of ``tutor aspects serialize``. The ``sql`` section of the dataset yaml can be moved to its own file in the `queries`_ directory and included in the yaml like so:
+   .. code-block:: bash
 
-.. code-block:: yaml
+      tutor aspects import_superset_zip ~/Downloads/your_file.zip
 
-   sql: "{% include 'openedx-assets/queries/query.sql' %}"
+4. Update database connection strings to use template variables.
+5. Validate and rebuild:
 
+   .. code-block:: bash
 
-However, please keep in mind that the assets declaration is itself a jinja template. That means that any jinja used in the dataset definition should be escaped. There are examples of how to handle this in the existing queries, such as `dim_courses.sql`_.
+      tutor images build aspects-superset --no-cache
+      tutor aspects check_superset_assets
+      tutor local do import-assets
 
-.. _queries: tutoraspects/templates/openedx-assets/queries/
+6. Submit a pull request with screenshots and details of your contributions.
 
-.. _dim_courses.sql: tutoraspects/templates/openedx-assets/queries/dim_courses.sql
+Release Workflow
+================
 
+Releases are handled via GitHub Actions:
 
-Releasing tutor-contrib-aspects
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Trigger the **Bump version and changelog** action to update the version and changelog.
+- Merge the PR to initiate the **release** and **build-image** workflows.
 
-Changelog, package version, PyPI release, and image building are all handled via manually triggered Githib Actions.
+Ensure the updated version appears on `PyPI <https://pypi.org>`_ and DockerHub.
 
-To trigger a build you must have access to manually trigger the "Bump version and changelog" action. This will update the version and changelog in a new PR. If the PR looks good, you can approve and merge it. Merging this PR will:
+Additional Resources
+=====================
 
-- Trigger the "release" workflow which will tag a Github release with the new version number, and then push the release to PyPI
-- Trigger the "build-image" workflow, which builds our images for aspects, aspects-superset, and openedx to the EduNEXT DockerHub repositories
+- `Tutor Documentation <https://docs.tutor.overhang.io>`_
+- `Aspects Beta Progress <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3861512203/Aspects+Beta>`_
```

Comment:
Can you links for Superset, dbt, event-routing-backends, and tracking logs docs?

Reply:
sure

Reply:
changes resolved in commit 263f568